movie-showcase/src/routes/managing/+page.svelte

```diff
@@ -0,0 +1,128 @@
+<script lang="ts">
+    import { movieArray } from "../../stores";
+    import { slideDuration } from "../../stores";
+    import { carouselItemCount } from "../../stores";
+
+    let title: string;
+    let year: string;
+
+    async function addToArray() {
+        const response = await fetch("/managing", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                title: title,
+                year: year,
+            }),
+        });
```

Comment:
I appreciate that you're making a call to your own server in order to hide the API_KEY from users 👍